### DISCUSSION TITLE
How do I change a button's background color?

### DISCUSSION THREAD

**Discussion by robkri** [bdlukaa/fluent_ui#917] [Q&A] [answered]
I'm trying to customize a `ToggleButton`'s background color when it is checked via the `FluentThemeData`.

I'm already defining some custom colors for light and dark theme like this:
```
FluentThemeData.light().copyWith(
  accentColor: AccentColor.swatch(const {"normal": Color.fromRGBO(65, 83, 229, 1)}),
  scaffoldBackgroundColor: Colors.white,
);
```
This works as expected. Now I'm trying to define a custom background color for `ToggleButton` when it is checked. Therefore i added 
```
toggleButtonTheme: ToggleButtonThemeData(
  checkedButtonStyle: ButtonStyle(backgroundColor: ButtonState.all(Colors.red)),
),
```
to the `copyWith` method's parameters.
Unfortunately this doesn't seem to do anything. Am I doing it correctly?

**Comment by bdlukaa** [comment]
Sorry, but I can not reproduce the issue.

Are you not confusing "ToggleButton" with "FilledButton"?

**Reply by robkri** [reply]
I seems to have been some local problem as today it works as expected. Although I really don't know what changed since last week.

Thank you for answering anyway.

**Comment by robkri** [comment, marked as answer]
Apparently there was some local problem last week as today it works. So to answer my own question: 

Yes, this is the correct way to change a `ToggleButton`'s background color.
